src/table/table.component.tsx

```diff
@@ -215,16 +240,31 @@ const Table = React.memo((props: TableProps): React.ReactElement => {
             })}
           </MuiTableHead>
           <MuiTableBody
-            sx={{ position: 'relative' }}
+            sx={{ position: 'relative', zIndex: 0 }}
             aria-describedby="table-loading-indicator"
             aria-busy={!loadedData}
           >
             {tableInstance.getRowModel().rows.map((row) => {
               return (
                 <MuiTableRow
-                  key={row.id}
+                  key={row.original._id}
                   sx={{ display: 'flex', flexDirection: 'row' }}
+                  selected={row.getIsSelected()}
+                  aria-checked={row.getIsSelected()}
                 >
+                  <MuiTableCell
+                    padding="checkbox"
+                    sx={{ width: '48px' }}
+                    scope="row"
+                  >
+                    <Checkbox
```

Comment:
These checkboxes look a little big/have too much padding imo - especially in a text only table. Can we experiment with size small checkboxes and reducing their padding (which reduced the size of the ripple effect)

Also, they aren't vertically aligned when an image or trace channel is present